kuma/api/v1/urls.py

```diff
@@ -12,6 +12,11 @@
         views.send_subscriptions_feedback,
         name="api.v1.send_subscriptions_feedback",
     ),
+    path(
+        "plus/landing-page/",
```

Comment:
```suggestion
        "plus/landing-page-variant/",
```

Reply:
I'm going to override that and make this change:
```diff
diff --git a/kuma/api/v1/urls.py b/kuma/api/v1/urls.py
index 8504daa8d..182652583 100644
--- a/kuma/api/v1/urls.py
+++ b/kuma/api/v1/urls.py
@@ -13,12 +13,12 @@ urlpatterns = [
         name="api.v1.send_subscriptions_feedback",
     ),
     path(
-        "plus/landing-page/",
+        "plus/landing-page/variant/",
         plus.landing_page_variant,
         name="api.v1.plus.landing_page_variant",
     ),
     path(
-        "plus/landing-page-survey/",
+        "plus/landing-page/survey/",
         plus.landing_page_survey,
         name="api.v1.plus.landing_page_survey",
     ),

```
Now they're more consistent and neater.

Reply:
Nice, I like that even better!